docs/agents/browser.md

```diff
@@ -0,0 +1,123 @@
+ 
+ - Hard page loads for traditional web applications as well as the single-page applications
+ - Soft page loads generated via the `routeChange` command for the single-page applications
+ - HTTP requests
+ - [Web application resources, such as images, CSS files, etc.](/experience/resources/)
+ - [Web vitals](/experience/webvitals/)
+ - [Element timing](/experience/element-timing/)
+ - [Long tasks](/experience/longtasks/)
+ - [User sessions](/experience/user-identification/)
+ - [On-page transactions](/experience/on-page-transaction/)
+
+![Sematext Experience Overview](../images/experience/overview.png)
+
+## Development
+
+The RUM script is developed using [ECMAScript 2015](https://en.wikipedia.org/wiki/ECMAScript) and uses a few tools to help with the build and working with the code. Those tools include:
+ 
+ - `npm` - [Node package manager](https://npm.org)
+ - `yarn` - [Package dependency manager](https://yarnpkg.com)
+ - `flow` - [Static type checker](https://flow.org)
+ - `Cypress.io` - [Integration tests framework](https://www.cypress.io)
+
+### The Architecture of the RUM Script
+
+There are two parts of the RUM script: 
+
+ - The loader script,
+ - The metrics gathering script itself.
+
+The loader script is responsible for loading the main, metrics gathering script itself and adds the actual `<script>` tags with the content on the main metric script. 
+
+The loader script will keep track and buffer any of the commands that need to be sent after the main metrics gathering script will be loaded and configured. When the main RUM script finally loads it will read any pending
+commands will be executed.
+
+### Working on the Loader Script
+
+The main entry point of the loader script is located in `src/snippet.js`. This is the unminified [ES2015]((https://en.wikipedia.org/wiki/ECMAScript)) version of the loader script which is responsible for loading the main RUM script. 
+
+To generate the minified snippet of the loader script use `yarn run generate-snippet`.
+
+### Working on the Main RUM Script
+
+The main entry point of the RUM script is located in `src/index.js`. Use `eslint` to catch any style issues and
+`flow` to catch any type errors.
+
+You can use the following commands:
+
+ - `yarn run start` to start the development server
+ - `yarn run lint` to run `lint` on the code
+ - `yarn run flow` to run `flow` type checks
+ - `yarn run build` to build the production bundle
+ - `yarn run dev` to build the development (not minified) bundle
+
+### Supported Commands
+
+The following commands are currently supported by the RUM script:
+
+ - `config` - Command used to configure the RUM script. It requires the `token` and `receiverUrl` configuration parameters. You can also provide the `ignoreBotData` Boolean parameter to ignore metrics coming from bots.   
```

Comment:
Since these are clearly commands, we can drop repetitive things like "Command used to..." and avoid needless verbosity.

Reply:
Good point, adjusted. Hopefully this looks better now.